config.test.yaml

```diff
@@ -39,6 +39,7 @@ templates:
       - mediawiki/v1/content
       - mediawiki_v1_graphoid
       - mediawiki/v1/mobileapps
+      - analytics_v1_pageviews
```

Comment:
I agree. For the second, though, I'd name it `/data/v1/stats/views.yaml` as their endpoints will likely end up as `/{domain}/v1/data/stats/views/...`